integration-test/pom.xml

```diff
@@ -64,6 +90,33 @@
 
     <dependency>
       <groupId>${project.groupId}</groupId>
+      <artifactId>livy-integration-test-minicluster-dependencies_2.10</artifactId>
```

Comment:
Is it safe to put both 2.10 and 2.11 jar in the same classpath, from my understanding it is hard to guarantee the method we called is coming from 2.10 rather 2.11.